packages/react-core/src/next/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+import { css } from '@patternfly/react-styles';
+import { Menu, MenuContent, MenuProps } from '../../../components/Menu';
+import { Popper } from '../../../helpers/Popper/Popper';
+
+export interface DropdownProps extends MenuProps {
+  /** Anything which can be rendered in a dropdown. */
+  children?: React.ReactNode;
+  /** Classes applied to root element of dropdown. */
+  className?: string;
+  /** Renderer for a custom dropdown toggle. Forwards a ref to the toggle. */
+  toggle?: (toggleRef: React.RefObject<any>) => React.ReactNode;
+  /** Flag to indicate if menu is opened.*/
+  isOpen?: boolean;
+  /** Function callback called when user selects item. */
+  onSelect?: (event?: React.MouseEvent<Element, MouseEvent>, itemId?: string | number) => void;
+  /** Callback to allow the dropdown component to change the open state of the menu that is set with the isOpen prop .
+   * If this is not provided the component will not close the menu when tab or escape are clicked. */
```

Comment:
Nit:

```suggestion
   * If this is not provided the component will not close the menu when tab or escape are pressed. */
```

Related more to the prop itself, I might be a bit wary about requiring the prop to be passed in in order to close the dropdown menu via escape/tab keys, especially since without the prop you can open/close the dropdown via mouse/Enter or Space key without issue. I don't think I would block over it if it'd require more of an investigation/tweaking another component, though, so maybe something to add as a to-do?